src/Compilers/CSharp/Portable/Symbols/TypeWithAnnotations.cs

```diff
@@ -728,7 +728,8 @@ public TypeWithAnnotations SetUnknownNullabilityForReferenceTypes()
         public override bool Equals(object other)
 #pragma warning restore CS0809
```

Comment:
@AlekseyTs Given that it's now implemented, should the obsolete attribute be removed?

Reply:
> Given that it's now implemented, should the obsolete attribute be removed?

No. The fact that it is implemented doesn't change our intent to prevent people to use it when we can detect that.